match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -159,9 +164,23 @@ def ensure_keys_file_exists(google_cloud_keys_file)
         fastlane_folder_gc_keys_path = File.join(FastlaneCore::FastlaneFolder.path, DEFAULT_KEYS_FILE_NAME)
         return fastlane_folder_gc_keys_path if File.exist?(fastlane_folder_gc_keys_path)
 
-        # User doesn't seem to have provided a keys file
-        UI.message("Looks like you don't have a Google Cloud #{DEFAULT_KEYS_FILE_NAME.cyan} file yet")
+        # User doesn't seem to have provided a keys file.
+        UI.message("Looks like you don't have a Google Cloud #{DEFAULT_KEYS_FILE_NAME.cyan} file")
```

Comment:
Right, you don't need both. I imagine there will be two possible setups here:

1) Teams that use Google Cloud heavily and everyone already has the Cloud SDK on their dev machines. These folks will want to specify google_cloud_project_id in their Matchfile and have developers use application default credentials locally, but specify google_cloud_keys_file when running match in their CI environment. This is the setup for which this PR is intended. 

2) Teams that don't expect everyone to have the Cloud SDK installed and just want to make it easy for developers to run builds locally on ad-hoc basis. These teams probably want to use match's cloud storage as it's currently designed, with every dev having their own keys on their workstation.

What do you think would be the best UI flow, in the case where google_cloud_project_id is specified but the Cloud SDK is absent or there's no application default keys present?

Reply:
I think we _could_ just move this line further down in the execute above https://github.com/fastlane/fastlane/pull/14031/files#diff-2492a283842f3bf95ee3a07a4cbd9c8eL169 🤔  I think its supposed to be tied with that block of code anyway since that block of code is the one that will help the user walk through creating a keys files.

Reply:
Done